main/Badge/index.tsx

```diff
@@ -24,8 +25,8 @@ const StyledView = styled.View`
   align-items: center;
   opacity: ${(props: badgeProps): number =>
     props.count === 0 ||
-    props.count! <= props.maximumValue! ||
-    !props.opacityVisible
+      props.count! <= props.maximumValue! ||
+      !props.opacityVisible
       ? 1
       : 0.6};
```

Comment:
I think it's better to move this part inside the component because the logic is complex.

Reply:
we think this logic will filter 2 conditions,
1. count is more than max count 
2. when count is more than max count, User`s option(Opacity Visible) will defined component`s opacity
but i dont know how to move this logic inside the component, not style code..

Reply:
It would be more readable to use conditional statements before `render`.